Documentation/creating-new-package.md

```diff
@@ -0,0 +1,29 @@
+# Creating new .NET IoT package
```

Comment:
This title is kind of ambiguous, it may be that users want to create a local package with their changes and might get confused and follow the bellow instructions while they can get what they want by just calling build.cmd. My suggestion would be to make it clear that these instructions are not for creating a package, but for releasing new versions to NuGet.